backend/spec/api-spec.yaml

```
@@ -0,0 +1,75 @@
+openapi: 3.0.1
```

yamllint

[error] wrong new line character: expected \n

(new-lines)